packages/atomic/src/components.d.ts

```diff
@@ -3165,6 +3174,8 @@ declare namespace LocalJSX {
          */
         "withDatePicker"?: boolean;
     }
+    interface AtomicIpxLayout {
```

Comment:
There is a duplicate interface above. Are both needed?
Also why is it empty? Don't you need at least a width or something similar.

Reply:
Thats autogenerated when building the project